content/blog/2022/07/vscode-camel-language-support-0.4.0-release/index.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "Released version 0.4.0 of VSCode Language Support for Apache Camel"
```

Comment:
The right naming is "VS Code" with a space (or "Visual Studio Code" but the long name is not used that much)